packages/http-cors/index.d.ts

```diff
@@ -3,6 +3,7 @@ import middy from '@middy/core'
 export interface Options {
   getOrigin?: (incomingOrigin: string, options: Options) => string
   credentials?: boolean | string
+  disableBeforePreflightResponse: true
```

Comment:
With `true` as the only valid input according to this interface, the latest update requires me to set this to true. It's not optional either as the documentation specifies. So shouldnt this be: `disableBeforePreflightResponse?: boolean`?

Reply:
fix releasing shortly